packages/connect-web/src/popup/index.ts

```diff
@@ -324,8 +324,7 @@ export class PopupManager extends EventEmitter {
             chrome.tabs.remove(this.popupWindow.id, () => {
                 _e = chrome.runtime.lastError;
             });
-        } else if (!this.settings.debug) {
-            this.popupWindow.close();
+        } else {
             this.popupWindow = null;
```

Comment:
This is still not clear to me why is in an `else` statement instead of 

```
        if (!this.popupWindow) return;
        if (this.settings.env === 'webextension') {
            // @ts-expect-error
            let _e = chrome.runtime.lastError;
            chrome.tabs.remove(this.popupWindow.id, () => {
                _e = chrome.runtime.lastError;
            });
        }
        this.popupWindow = null;
```

Reply:
Why not setting `this.popupWindow = null` when in `webextension` ?